worker/workercmd/guardian.go

```diff
@@ -152,3 +177,64 @@ func detectGuardianFlags(logger lager.Logger) []string {
 func flagify(env string) string {
 	return strings.Replace(strings.ToLower(env), "_", "-", -1)
 }
+
+func getGdnFlagsFromConfig(configPath string) ([]string, error){
+	var configFlags []string
+
+	if configPath != "" {
+		file, err := os.Open(configPath)
+		if err != nil {
+			return []string{}, err
+		}
+		defer file.Close()
+		bs := bufio.NewScanner(file)
+		for bs.Scan() {
+			line := bs.Text()
+
+			if len(line) == 0 || line[0] == '#' {
```

Comment:
Maybe we can also add a case for the semicolon being a comment line?

Reply:
Good catch! Done.